src/group_theory/submonoid/membership.lean

```diff
@@ -265,3 +265,28 @@ attribute [to_additive add_submonoid.multiples_eq_closure] submonoid.powers_eq_c
 attribute [to_additive add_submonoid.multiples_subset] submonoid.powers_subset
 
 end add_submonoid
+
+lemma of_mul_image_powers_eq_multiples_of_mul {x : M} :
```

Comment:
At the moment these are not used but I thought they might be useful in removing the `congr`.